static/styles/night_mode.css

```diff
@@ -342,6 +342,10 @@ body.night-mode {
         border-color: transparent;
     }
 
+    .compose_control_buttons_container .divider {
+        color: hsla(0, 0%, 100%, 0.5);
```

Comment:
Should this be `0, 0, 100?  Presumably we want something that's designed to align with the normal night theme font color: `color: hsl(236, 33%, 90%);`

That said, if the goal is to make the thing grey, I think if we just do `opacity: 0.7` or similar in the original CSS, that should transparently work in both themes (and is our usual way to avoid adding new colors to the app for new components)

Reply:
Actually, this is the same as the dividers in the navbar. So, it is fine?

Reply:
Yeah; I'd say it's a design pattern mistake bug that the navbar elements use their own custom color, but in may be simpler to clean that up in a separate PR since we have enough changes happening in this one already :).